orttraining/orttraining/python/training/ortmodule/_utils.py

```diff
@@ -50,6 +51,33 @@ def _ortvalue_to_torch_tensor(ortvalue, device):
     dlpack_tensor = ortvalue.to_dlpack()
```

Comment:
Since you replace all _ortvalue_to_torch_tensor by _ortvalues_to_torch_tensor, maybe we can remove the old one.

Reply:
I let it just in case somebody needs to convert only one ortvalue. I can remove it.